backend/app.py

```diff
@@ -1,19 +1,29 @@
 from jina.types.document.generators import from_csv
 from jina import DocumentArray, Flow
 from my_executors import ProtBertExecutor, MyIndexer
+from backend_config import protein_path
+from random import randint
+
 
 def main():
     proteins = DocumentArray(
         from_csv(
-            fp=open("data/samelength.csv"),
+            fp=open(protein_path),
```

Comment:
Here we should use a context manager, as @gmelodie mentioned at the Jina maintainers meeting.